typing/parmatch.ml

```diff
@@ -42,7 +42,206 @@ let rec omegas i =
 
 let omega_list l = List.map (fun _ -> omega) l
 
-let zero = make_pat (Tpat_constant (Const_int 0)) Ctype.none Env.empty
+module Pattern_head : sig
```

Comment:
I don't know what this diff is doing in this PR, but whatever.

Reply:
Yeah, this PR is not based on a tagged branch, so it includes some extra commits...
See:
![rematch](https://user-images.githubusercontent.com/118852/61211307-2c303300-a6f7-11e9-94df-efd65f1863f2.png)